lib/rust/ensogl/core/src/display/scene/layer.rs

```diff
@@ -780,6 +776,31 @@ impl LayerModel {
         layer
     }
 
+    /// Create a virtual sublayer of this layer. The order of virtual layer creation determines
+    /// depth ordering of their content objects of the type specified in the virtual layers' type
+    /// parameter.
+    pub fn create_virtual_sublayer<S: Shape>(
+        self: &Rc<Self>,
+        _name: impl Into<String>,
+    ) -> VirtualLayer<S> {
+        let system_id = ShapeSystem::<S>::id();
+        let mut partitions = self.virtual_sublayer_buffer_partitions.borrow_mut();
+        let index = partitions.entry(system_id).or_default();
+        let id = VirtualLayerId { index: *index };
+        *index += 1;
+        VirtualLayer { layer: WeakLayer { model: self.downgrade() }, id, shape: default() }
+    }
+
+    /// Create a virtual sublayer of this layer. The virtual layer will manage objects of the same
+    /// type as the provided parameter; the value is not used.
```

Comment:
"the value is not used" - so why do we provide it here? Instead, we can call this function using super fish operator, right?